sdk/identity/identity/rollup.base.config.js

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Did you try using the dev-tool's shared rollup config instead? You can find an example in text analytics.

Reply:
I'll take a look!

Reply:
It doesn't seem to work for browser tests 🤔 when I run browser tests after I move to dev-tool's rollup makeConfig, 0 browser tests get executed. I could be missing something

Reply:
Huh weird, Did you check whether karma.config.js points to the right dist file?

Reply:
We can log a separate tracking item to onboard to the shared rollup if the initial attempt saw any blockers

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/16535

Reply:
Unresolving this thread to clarify: identity is _already_ migrated to the shared rollup configuration. See [`rollup.config.js`](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/rollup.config.js).

Reply:
Interesting, so why this file is being checked in?

Reply:
Because of the browser tests.

Reply:
The shared rollup seems to not work for browser tests? I could be missing something!

Reply:
I made it work! I’ll switch to the shared config.

Reply:
Closed: https://github.com/Azure/azure-sdk-for-js/issues/16535